src/jit/compiler.h

```diff
@@ -9405,6 +9391,523 @@ inline LclVarDsc::LclVarDsc(Compiler* comp)
 {
 }
 
+template <typename TVisitor>
```

Comment:
This deserves a header comment, e.g. restrictions on `DoLclVarsOnly` and the walk order.